index.html

```diff
@@ -12,34 +13,42 @@
 
   <body>
     <header>
-      <section>
-        <h1>Bronwyn Detlofsen</h1>
-        <h5>Web Developer</h5>
-      </section>
-      <nav>
-        <ul id="navigation">
-          <li class="nav" data-category="skills"><a href="#">Skills</a></li>
-          <li class="nav" data-category="projects"><a href="#">Projects</a></li>
-          <li class="nav" data-category="experience"><a href="#">Experience</a></li>
-          <li class="nav" data-category="interests"><a href="#">Interests</a></li>
-          <li class="nav" data-category="profile"><a href="#">Profile</a></li>
-        </ul>
-      </nav>
+      <div>
+        <section>
+          <h1>Bronwyn Detlofsen</h1>
+          <h5>Web Developer</h5>
+        </section>
+        <nav>
+          <ul id="navigation">
+            <a href="#"><li class="nav" data-category="skills">Skills</li></a>
+            <li class="nav" data-category="projects"><a href="#">Projects</a></li>
+            <li class="nav" data-category="experience"><a href="#">Experience</a></li>
+            <li class="nav" data-category="interests"><a href="#">Interests</a></li>
+            <li class="nav" data-category="profile" id="lastli"><a href="#">Profile</a></li>
+          </ul>
+        </nav>
+    </div>
     </header>
 
     <main>
 
       <section class="category" id="skills">
         <h2>Skills</h2>
-          <p>JavaScript, HTML, CSS, jQuery</p>
+          <p>JavaScript, HTML, CSS, jQuery, Handlebars (templating), JSON</p>
       </section>
 
       <section class="category" id="projects">
         <h2>Projects</h2>
-          <h3>Totally Rad Hangman</h3>
-              <ul>
-                <li>Web-based hangman game written entirely in vanilla <em>JavaScript</em> and using <em>HTML</em> and <em>CSS</em>.</li>
-      </section>
+        <article id="projblurb">
+        </article>
+          <script id="project-template" type="text/x-handlebars-template">
```

Comment:
I'm surprised this script is in the head, but I like that it still works just fine. Any reason to not move it from the head? I actually almost like it better NOT in the head, since it makes it easier to know where it is supposed to go.